target/linux/ath79/image/generic.mk

```diff
@@ -903,6 +903,24 @@ define Device/netgear_wndr3800
 endef
 TARGET_DEVICES += netgear_wndr3800
 
+define Device/netgear_wndrmac
+  $(Device/netgear_wndr3700-v2)
```

Comment:
Please use $(Device/netgear_wndr3x00) here.

Reply:
If I pull `$(Device/netgear_wndr3x00)` here I'd have to redefine everything `$(Device/netgear_wndr3700-v2)` does, which I don't want to do. I'm technically *relying* on the predefines.

Reply:
(Will do as is being requested)

Reply:
This is essentially one of the lessons ar71xx has told: As you see with wndr3xxx/wndrmac devices in ar71xx, you have to go through up to four definitions for some devices to really find all of the variables set. That's quite inconvenient and misleading.
In ath79, in contrast, devices are not derived from each other, but just from specifically created shared nodes. While this might cause some lines to be duplicated, it's actually much easier to read and understand.